.github/workflows/test-suite.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     name: "Python ${{ matrix.python-version }} ${{ matrix.os }}"
     runs-on: "${{ matrix.os }}"
     strategy:
+      fail-fast: false
```

Comment:
The [GitHub Actions `jobs.<job_id>.strategy.fail-fast` setting](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) controls how failures are handled. The default of `true` will fail the entire job when any job in the matrix fails.

With the introduction of platform-specific worker testing, subprocesses, and operating system signal handling, there is potential for specific workflow jobs to occasionally fail. In these cases, it can help to set `fail-fast: false` to ensure that those specific failed jobs can be identified.

Reply:
Is this still needed, given that you know why the tests are flaky?

Reply:
Is it _needed_? No. Is it _helpful_? I think so. But if you don't then I will remove it (b1ab28c).

Reply:
...and now we're seeing a bunch of skipped jobs because of that failing websocket test (https://github.com/encode/uvicorn/pull/1995#issuecomment-1575657201).